specutils/io/default_loaders/tests/test_jwst_reader.py

```diff
@@ -366,7 +366,7 @@ def generate_s3d_wcs():
 
 
 @pytest.fixture()
-def tmp_asdf(tmpdir):
+def tmp_asdf(tmp_path):
```

Comment:
But why? It looks unused.

```suggestion
def tmp_asdf():
```

Reply:
Hm, it does indeed 🤔. I'm surprised codestyle didn't catch that.

Reply:
Some fixtures do magic behind the scenes but I think tmp_path is not needed unless it is explicitly used?